.travis.yml

```diff
@@ -3,7 +3,14 @@ language: ruby
 rvm:
   - "2.3.7"
 
-dist: precise
+dist: trusty
+
+jobs:
+  include:
+    - script: bundle exec rake
+    - script: cd lara-plugin-api-V2 && npm i && npm run test
+    - script: cd lara-typescript && npm i && npm run test
+    - script: ./script/cypress-on-travis.sh || travis_terminate 1;
```

Comment:
This has a nice side effect of running all the tests in parallel.  Without the Cypress part it cuts about 1 minute from the test run.